kimhaejung/app.js

```diff
@@ -52,6 +56,59 @@ app.post("/users/signup", async (req, res, next) => {
   res.status(201).json({ message: "userCreated" });
 });
 
-app.listen(port, function () {
-  console.log(`server listening on port ${port}`);
+//
+app.patch("/posts/:userId/:postId", async (req, res, next) => {
```

Comment:
- 현재 엔드포인트 구조를 path parameter로 잘 잡아주셨는데, 리소스의 상하 관계가 다소 모호한 것 같습니다. 
- posts다음에 따라오는 id 값이 postId이어야지 올바른 리소스의 상하 관계를 드러냈다고 표현할 수 있겠네요!

```suggestion
app.patch("/posts/:postId", async (req, res, next) => {
```

Reply:
- 추가로 userId 와 같이 유저 정보를 담는 내용은 보안에 민감한 사항일 수 있으니, request.body에 담아서 보내는 것을 권장드립니다.